xml/System/Environment.xml

```diff
@@ -1753,13 +1753,20 @@ To identify the operating system platform, for example, Linux or Windows, you ca
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the number of processors on the current machine.</summary>
-        <value>The 32-bit signed integer that specifies the number of processors on the current machine. There is no default. If the current machine contains multiple processor groups, this property returns the number of logical processors that are available for use by the common language runtime (CLR).</value>
+        <summary>Gets the number of processors available to the current process.</summary>
+        <value>The 32-bit signed integer specifies the number of processors that are available.</value>
         <remarks>
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
- For more information about processor groups and logical processors, see [Processor Groups](/windows/win32/procthread/processor-groups).
+On Linux and macOS systems for all .NET versions and on Windows systems starting with .NET 6, this API returns the minumum of:
+- The physicial number of CPUs on the machine 
```

Comment:
The number of logical processors.  There is no such thing as a "physicial number".

Reply:
```suggestion
- The physical number of CPUs on the machine 
```